docs/scos/user/technology-partners/202108.0/payment-partners/bs-payone/legacy-demoshop-integration/payone-facade.md

```diff
@@ -14,42 +14,45 @@ redirect_from:
-| `isPaymentAppointed` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from Payone with "Appointed" status for a certain order. |
-| `isPaymentOther` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from Payone with status different from "Paid", "Underpaid" or "Appointed" for a certain order. |
-| `isPaymentCapture` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from Payone with "Capture" status for a certain order. |
-| `postSaveHook` | `QuoteTransfer`, `CheckoutResponseTransfer` | `CheckoutResponseTransfer` | Handles redirects and errors after order placement. |
-| `getPaymentLogs` | `ObjectCollection` | array of `PayonePaymentLogTransfer` | Gets payment logs (both api and transaction status) for specific orders in chronological order. |
+| `isPaymentNotificationAvailable` | Order Id, Order Item Id | bool | Checks if there are unprocessed transaction status updates from PayOne for a certain order. |
+| `isPaymentPaid` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Paid" status and balance is zero or negative for a certain order. |
+| `isPaymentOverpaid` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Paid" status, and the balance is negative for a certain order. |
+| `isPaymentUnderpaid` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Underpaid" status for a certain order. |
+| `isPaymentRefund` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Refund" status for a certain order. |
+| `isPaymentAppointed` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Appointed" status for a certain order. |
+| `isPaymentOther` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the status different from "Paid", "Underpaid" or "Appointed" for a certain order. |
+| `isPaymentCapture` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Capture" status for a certain order. |
+| `postSaveHook` (deprecated) | `QuoteTransfer`, `CheckoutResponseTransfer` | `CheckoutResponseTransfer` | Handles redirects and errors after an order is placed. |
+| `orderPostSave` | `QuoteTransfer`, `CheckoutResponseTransfer` | `CheckoutResponseTransfer` | Handles redirects and errors after an order is placed. Depending on the payment method, it executes the authorization or pre-authorization API call. Updates `CheckoutResponseTransfer` with errors or/and redirect URL accordingly to the API response. |
```

Comment:
Consider "After an order has been placed", instead of "it executes" -> just executes. Also for "or/and" - avoid using slashes like that: https://developers.google.com/style/slashes?hl=en